cns/middlewares/k8sSwiftV2.go

```diff
@@ -28,6 +28,7 @@ var (
 	errGetMTPNC                 = errors.New(NetworkNotReadyErrorMsg + " - failed to get MTPNC")
 	errInvalidSWIFTv2NICType    = errors.New("invalid NIC type for SWIFT v2 scenario")
 	errInvalidMTPNCPrefixLength = errors.New("invalid prefix length for MTPNC primaryIP, must be 32")
+	errDeleteTimestampSet       = errors.New("MTPNC deletion timestamp is set")
```

Comment:
deletion timestamp being set is an implementation detail. the important thing is that the object is deleting.